apps/web-portal/lib/WebPortalExpressApp.mjs

```diff
@@ -44,8 +45,14 @@ export class WebPortalExpressApp {
   }
 
   startAndAttach() {
-    this.#http.listen(this.#config.http.port, () => {
-      console.log('Server started')
+    this.#app = this.#http.listen(this.#config.http.port, () => {
+      logger.info('Server started')
     })
   }
+
+  stop() {
```

Comment:
Good catch, we would need that one for integration tests propose, plz add jsdocs

Reply:
done

Reply:
it's not a property, define jsdocs as for method (see comment above)